yogstation/code/modules/vending/wardrobes.dm

```diff
@@ -101,6 +101,7 @@
 					/obj/item/clothing/mask/bandana/red = 3,
 					/obj/item/clothing/suit/armor/vest/secmiljacket = 3,
 					/obj/item/clothing/shoes/yogs/namboots = 5,
+					/obj/item/clothing/gloves/yogs/namgloves = 5,
```

Comment:
Why do you always try to sneak in stuff not related to the PR?

Reply:
well I made the juggernaut gloves then realized wait a second the nam suit doesnt have any gloves so I gave it gloves